apps/web/views/ascents/create.rb

```diff
@@ -1,11 +1,8 @@
-require_relative './create_methods'
-
 module Web
   module Views
     module Ascents
       class Create
         include Web::View
-        include Web::Views::Ascents::CreateMethods
```

Comment:
If you put all logic to ascents/_from, it will not be reusable(we would like to use the same form in edit/update actions as in the users#edit. Form for these actions requires different path, submit label, etc.)

Reply:
The general principle is: implement something when the requirement comes, not before. The form will change drastically in the future, so it doesn't make sense to make decisions now. If you want to have a different path and label you can pass them as locals to the partial. But I guess we will use some kind of FormObject in the future.